pypy/module/cpyext/parse/cpyext_object.h

```diff
@@ -149,12 +149,24 @@ typedef struct {
     objobjargproc mp_ass_subscript;
 } PyMappingMethods;
 
-/* Result of calling PyIter_Send */
-typedef enum {
+/* Result of calling PyIter_Send
+
+Note: small deviation from CPython definitions here:
+PyPy wants to return and int from "sendfunc" because it knows sizeof(int)
+and doesn't know sizeof(enum PySendResult).
+Therefore introduce the symbols like PYGEN_RETURN into the global namespace
+with an anoymous enum, but define PySendResult as int so that funnctions
```

Comment:
```suggestion
PyPy wants to return an int from "sendfunc" because it knows sizeof(int)
and doesn't know sizeof(enum PySendResult).
Therefore introduce the symbols like PYGEN_RETURN into the global namespace
with an anoymous enum, but define PySendResult as int so that functions
```